src/humanPose/HumanPoseAnalyzer.js

```diff
@@ -1096,6 +1096,14 @@ export class HumanPoseAnalyzer {
             clone.renderer.markMatrixNeedsUpdate();
         });
     }
+    
+    getDataByTimestamp(timestamp) {
```

Comment:
This feels mostly unnecessary and could be replaced by getClonesByTimestamp without losing much. `undefined` vs `[]` isn't a big difference.